x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/trace_waterfall_context.tsx

```diff
@@ -79,6 +97,16 @@ export function TraceWaterfallContextProvider({
   const { duration, traceWaterfall, maxDepth, rootItem, legends, colorBy } = useTraceWaterfall({
     traceItems,
   });
+  const [warningMessage, setWarningMessage] = useState(() => {
+    const hasBeenDismissed = DISMISSED_WARNINGS.has(waterfallId);
+
+    return !(rootItem || hasBeenDismissed) ? FALLBACK_WARNING : undefined;
+  });
+
+  const dismissWarning = () => {
```

Comment:
I honestly don't see any reasons why to dismiss this message. It will be either the warning message or the waterfall.